daqWheel/include/pinDefs.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+//THESE ARE FILLER DEFINITIONS UNTIL PCBA IS DESIGNED
+#define WHEEL_SPEED_SENSOR_PIN 1
+#define BRAKE_TEMPERATURE_SENSOR_PIN 1
+#define SUSPENSION_POSITION_SENSOR_PIN 1
+
+#define BRAKE_TEMPERATURE_SCALAR 1
+#define BRAKE_TEMPERATURE_OFFSET 0
+
+#define SUSPENSION_POSITION_SCALAR 1
+#define SUSPENSION_POSITION_OFFSET 0
+
+#define FL_CAN_FRAME_ADDRESS 0x400
+#define FR_CAN_FRAME_ADDRESS 0x401
+#define BL_CAN_FRAME_ADDRESS 0x402
+#define BR_CAN_FRAME_ADDRESS 0x403
```

Comment:
I know one of the things we discussed initially was potentially having some kind of board class that would have all of these pieces of information abstracted away (eg. pin defs). Is that still something we want to do? Or are we ok with just using a pin defs / hardware defs file and moving forward

Reply:
Oh also, it would probably be good to change the pins so they're all reading different pins that are actually available on the esp32 so someone new could theoretically at least try to see if the code works without changing it, since right now they all map to the same pin (pin 1).